pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -2440,392 +2373,12 @@ class ShapePropagator : public PropertyPropBase {
     }
     return false;
   }
-
-  bool PropagateCompleteShapeOnNode(
```

Comment:
I'm not sure it's safe to delete this function, this function deal with the static shape , do you do some tests on typical models, such as bert ?

Reply:
Eventually, we will not use static shapes directly. But elaborate partial shapes carefully in dynamic shape scenarios. 

I will test them against our PyTorch/inference examples once it ready. And merge this PR only when the examples tests pass.

Reply:
The PR had success in running the BERT example with torch==1.7.1+cu110. The results is as following:
```
Naive PyTorch.
average time in 100 iterations: 0.045059432983398436 seconds
BladeDISC Optimization.
average time in 100 iterations: 0.006616170406341553 seconds
```